UnrealPlugin/ThirdPersonMPLocalDeploy.md

```diff
@@ -0,0 +1,118 @@
+# ThirdPersonMP Example Project Local Deployment and Debugging
+
+The purpose of this guide is to demonstrate running your game-server on your local machine, in a MPS-compliant way, so you can test and debug your server before uploading it to PlayFab.
+
+## Goals
+
+* Test the local deployment options for the ThirdPersonMP+GSDK project.
+* Verify your server executes properly when run using LocalMultiplayerAgent
+* Verify you can attach a debugger to your server instance
+
+## Requirements
+
+* Download Visual Studio. The [community version](https://visualstudio.microsoft.com/vs/community/) is free.
+	* Required workloads: .NET desktop development and Desktop development with C++
+* Download Unreal Engine Source Code. This plugin was tested on Unreal Engine 4.26.2. For instructions, see [Downloading Unreal Engine Source code (external)](https://docs.unrealengine.com/4.26/en-US/ProgrammingAndScripting/ProgrammingWithCPP/DownloadingSourceCode/).
+* [Completed Unreal Project](ThirdPersonMPSetup.md) with [PlayFab Unreal GSDK](ThirdPersonMPGSDKSetup.md) installed and configured
+* ["Development Server"](ThirdPersonMPBuild.md) configuration of your project built from Visual Studio
+* [Optional] Download the [LocalMultiplayerAgent](https://github.com/PlayFab/MpsAgent/releases)
+	* [Optional] Alternatively, download [LocalMultiplayerAgent source](https://github.com/PlayFab/MpsAgent/tree/main/LocalMultiplayerAgent).
+* "Debug" or "Release" configuration of LocalMultiplayerAgent built from this repo with Visual Studio
+	* Load [MpsAgent.sln](https://github.com/PlayFab/MpsAgent/blob/main/MpsAgent.sln) in Visual Studio, select configuration, build LocalMultiplayerAgent project
```

Comment:
* Open [MpsAgent.sln] (https://github.com/PlayFab/MpsAgent/blob/main/MpsAgent.sln) in Visual Studio. Select either debug or release configuration and then build the LocalMultiplayerAgent project.